graphql/concrete_notification/fields.ts

```diff
@@ -40,10 +41,13 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return await prisma.notification.findUnique({ where: { id: concreteNotification.notificationID } });
     }
 
-    @FieldResolver((returns) => NotificationMessage)
+    @FieldResolver((returns) => NotificationMessageType)
```

Comment:
What happens when a Notification does not have a message maintained?

Reply:
fixed